azure-ts-dynamicresource/package.json

```diff
@@ -0,0 +1,13 @@
+{
+    "name": "azure-ts-appservice",
+    "version": "1.0.0",
+    "devDependencies": {
+        "@types/node": "^10.3.3"
+    },
+    "dependencies": {
+        "@azure/arm-cdn": "^4.2.0",
+        "@azure/ms-rest-nodeauth": "^1.1.1",
```

Comment:
Shall we set the versions to the ones that don't cause the error?

Reply:
Done.